backend/Squirrel.ConsoleApp.BL/Services/Abstract/BaseDbService.cs

```diff
@@ -59,8 +56,8 @@ private QueryResultTable BuildTable(DbDataReader reader)
             var row = new List<string>(fieldCount);
             for (int i = 0; i < fieldCount; i++)
             {
-                var value = reader.IsDBNull(i) ? nullValue : reader[i].ToString();
-                row.Add(value ?? nullValue);
+                var value = reader.IsDBNull(i) ? string.Empty : (reader[i].ToString() ?? string.Empty);
```

Comment:
we added a `null` string `to display 'null' in empty cells in the results table on UI`

Reply:
It breaks mapping logics on Load latest changes